### DISCUSSION TITLE
Custom indentation column

### DISCUSSION THREAD

**Discussion by cfclavijo** [topiary/topiary#679] [Q&A] [answered]
Hello, all! Maybe the Title is not as descriptive as I wanted, but here is the thing. I'm working on adding Erlang to topiary and stumbled upon a particularity we have when exporting functions in Erlang (which is not crucial but would be much appreciated)
```
-export([ sum/2
        , sum/1
        , multiply/2
        ]).
````
as you can see, the indentation column is defined by the open square bracket.
Is it there a way to define this Indentation column?

Thanks!

And thank you for your effort on this project!

**Comment by Xophmeister** [comment, marked as answer]
Hi, @cfclavijo -- thanks for your question...and it's great you're working on Erlang support :pray: 

Currently, what you describe cannot be done in Topiary; it only supports basic indentation levels. However, this is an often-discussed issue -- see issue #170 for the details -- so it is certainly on our radar. There's no plan in the immediate future to implement this, but its increasing interest does push the needle in its favour!
